### PR TITLE
(chore) Move react-accordion, react-divider to react-components subfolder 

### PR DESCRIPTION
**Pull request by TristanWatanabe**
## Changes

- Moves `@fluentui/react-accordion`, and `@fluentui/react-divider` to react-components common folder using `move-packages` nx generator.

## Related Issue(s)

Part of #22427

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/Button-mount_base.html">554</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/Button-mount.html">553</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/Button-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-mount_base.html">288</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-mount.html">304</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">237</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">233</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">306</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">328</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/Avatar-mount_base.html">901</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/Avatar-mount.html">901</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/Button-mount_base.html">554</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/Button-mount.html">553</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/Button-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProvider-mount_base.html">1890</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProvider-mount.html">1786</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-mount_base.html">288</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-mount.html">304</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">237</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">233</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">306</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">328</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/MakeStyles-mount_base.html">1556</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22827/perf-test-react-components/MakeStyles-mount.html">1545</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                 |         Size (minified/GZIP) |
| ------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox</abbr>  | `60.564 kB`<br />`20.751 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr>           | `29.382 kB`<br />`10.047 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr> |   `13.61 kB`<br />`5.462 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>        |  `25.312 kB`<br />`8.202 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>  |   `20.672 kB`<br />`7.07 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/8437eb3b6023bdac5b234538e1116b1c7d24f53f'>8437eb3b6023bdac5b234538e1116b1c7d24f53f</a></sub>
<div id="bundleSizeComment9423"/>

**Review by ling1726 — APPROVED** [review]

**Review by khmakoto — APPROVED** [review]

**Comment by TristanWatanabe** [comment]
Pulling `react-badge` out from this move because of various active PRs involving it

File: packages/react-components/react-components/.storybook/main.utils.test.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ describe(`main utils`, () => {
       expect(actual).toEqual(expect.arrayContaining(expected));
 
       const first = actual[0];
-      expect(first.startsWith('../../../react-')).toBeTruthy();
+      expect(first.startsWith('../../react-')).toBeTruthy();
```
[the remark this answers is not in the document]
Comment (Hotell) [reply packages/react-components/react-components/.storybook/main.utils.test.js:16]: shame on me :D not the most robust test indeed but there were not many options. ty

**Review by Hotell — APPROVED** [review]

**Pull request merged**